.github/bot-pr-base.sh

```diff
@@ -41,6 +41,13 @@ HEAD_URL="https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/$HEAD_REPO"
 
 JOB_URL="https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
 
+bot_delete_comments_matching() {
+  COMMENTS=$(api_get "$ISSUE_URL/comments" | jq -er '.[] | select((.user.login == "github-actions[bot]") and (.body | startswith('"\"$1\""'))) | .url')
```

Comment:
Maybe store `$1` into a variable to understand what it's doing? Like:
```suggestion
  local search_matching=$1
  COMMENTS=$(api_get "$ISSUE_URL/comments" | jq -er '.[] | select((.user.login == "github-actions[bot]") and (.body | startswith('"\"$search_matching\""'))) | .url')
```